src/redux/walletconnect.js

```diff
@@ -601,7 +601,10 @@ export const walletConnectDisconnectAllByDappUrl = dappUrl => async (
 ) => {
   const { walletConnectors } = getState().walletconnect;
   const matchingWalletConnectors = values(
-    pickBy(walletConnectors, connector => connector.peerMeta.url === dappUrl)
+    pickBy(
+      walletConnectors,
+      connector => connector?.peerMeta?.url === dappUrl || !connector?.peerMeta
```

Comment:
added `!connector?.peerMeta` as a condition to remove the connector in bad state